File: 800/1360A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int a = sc.nextInt();
            int b = sc.nextInt();
            
            int minSide = Math.min(a, b);
            int maxSide = Math.max(a, b);
 
            int s = Math.max(2 * minSide, maxSide);
            
            System.out.println(s * s);
        }
    }
}

File: 800/1360A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int a, b;
        cin >> a >> b;

        int minSide = min(a, b);
        int maxSide = max(a, b);

        int s = max(2 * minSide, maxSide);

        cout << s * s << "\n";
    }
    return 0;
}
